docs/content/spark/sql-write.md

```diff
@@ -248,7 +248,7 @@ val stream = df
   .outputMode("append")
   .option("checkpointLocation", "/path/to/checkpoint")
   .format("paimon")
-  .start("/path/to/paimon/sink/table")
+  .toTable("${database}.${table}")
```

Comment:
modify schema evolution too.

And maybe we can add a line to old path style, just like:
```
.toTable("${database}.${table}")
// .start("/path/to/paimon/sink/table")
```